docs/product/releases/release-details.mdx

```diff
@@ -64,3 +64,15 @@ Releases are global per organization, and this list shows you other projects wit
 
 For more information about Sentry's mobile features, see the full documentation for [Android SDK](/platforms/android/), [iOS SDK](/platforms/apple/guides/ios/) and
 [React Native SDK](/platforms/react-native/).
+
+## Quick Release Details
```

Comment:
feels buried at the bottom of the page. 

It might make sense on `/product/releases/` near the "Track Release Health](https://sentry-docs-6hkk4ynfq.sentry.dev/product/releases/#track-release-health)" section, but that section needs some love to point people towards Insights first imo.

[Release Health](https://sentry-docs-6hkk4ynfq.sentry.dev/product/releases/health/) is also it's own page, which needs the same kind of love... maybe the Quick release stuff should be it's own page because it's really like some kind of 'Release Data Throughout the Product'

Reply:
accepted so we can iterate!

Reply:
Having it be its own page would be helpful for adding the bubbles part of it as I was struggling to think of a good place for that.